.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
     - $HOME/.gradle/wrapper/
 
 # Enabling container based infrastructure hoping it will help the build speed <= this didn't work well, so it's reverted
-# see http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+# see https://docs.travis-ci.com/user/migrating-from-legacy/
```

Comment:
@divyansh-gupta Can you add `https://docs.travis-ci.com/user/ci-environment` as well ?